codes/quantum/qubits/gauge_stabilizer.yml

```diff
@@ -4,15 +4,71 @@
 #######################################################
 
 code_id: gauge_stabilizer
-#physical: bits
-#logical: qubits
+physical: qubits
+logical: qubits
 
-name: 'Gauge stabilizer code'
-#introduced: '\cite{arxiv:2004.07935}'
+name: 'Gauge Stabilizer Codes'
 
```

Comment:
FYI You didn't add that description from before here.

Reply:
Yeah sorry, I got rid of it because I thought there was an error in it (and there was) and I wanted to make sure it was the only error in all of my commits. I will re add the description right now.